### ISSUE TITLE
Slot time label invisible when minTime starts out of alignment

### ISSUE THREAD

**Issue by arshaw** [fullcalendar/fullcalendar#2786]
Originally reported on Google Code with ID 2521

```
I found a bug, the row header with the time (in agendaWeek view) disappears
if you set the minTime to let's say  "08:30" and slotInterval to "00:60"

Link to ilustrate the problem:
https://jsfiddle.net/7ttnwvw5/
change minTime to 08:00 work
change minTime to 08:30 bug (time disappears)

I also attached a image to explain the problem.
```

Reported by `adriano.skroch` on 2015-06-15 15:19:40

<hr>
- _Attachment: fcbug.png<br>![fcbug.png](https://storage.googleapis.com/google-code-attachments/fullcalendar/issue-2521/comment-0/fcbug.png)_

Imported with **2** stars.

**Comment by idan89** [comment]
Hey there, 
I have found the same bug but with the following configuration:

minTime: "08:15:00",
maxTime: "22:15:00",
slotDuration: "00:30:00",

If you change slotDuration to "00:20:00" it works (slotDuration appears)
It also works if you change the minTime to let's say "08:00:00"

Evidence: https://jsfiddle.net/7ttnwvw5/1/

**Comment by adrianolsk** [comment]
@idan89 I didn't check this post for a while, but I manage to do a workaround,

take a look here https://jsfiddle.net/7ttnwvw5/2/
I create a function to fill the time header, it´s not beautiful but works
```js
 function TimeFix(durationInMinutes, minTime) {
        var hour = moment(minTime, "HH:mm");
        $(".fc-body .fc-slats table tr").each(function (index) {
            $(this).find("td.fc-widget-content").eq(0).html("<span>" + hour.format("HH:mm") + "</span>")

            hour.add(durationInMinutes, "minutes");

        });
        $(".fc-widget-header th").eq($(".fc-today").index()).addClass("fc-head-today").append("<div></div>");
    }
```
and I call it on viewRender event
```js
 viewRender: function (view, element) {
			TimeFix(30, "08:15:00");                   
}
```

Hope it help you.

**Comment by idan89** [comment]
Great! Thank you very much @adrianolsk :)
Cheers,

**Comment by arshaw** [comment]
@WilliamPeralta's PR: https://github.com/fullcalendar/fullcalendar/pull/3075

**Comment by guidosl** [comment]
+1

**Comment by adamduren** [comment]
Ran into this bug within five minutes of "getting started" with this library. Pretty big blocker for my particular application. Otherwise awesome library!

**Comment by arshaw** [comment]
fix queued up for next release

**Comment by HugoOliveiraThor** [comment]
@adrianolsk It works perfectly thanks i have a minTime "08:00:00" and my interval it was "00:25:00" and with that my time disapear and now with the TimeFix its works perfect !! Thank you very much you save my day !

**Comment by adamduren** [comment]
Can confirmed that this is fixed on master. Any timeline on the next release?

**Comment by adamduren** [comment]
Actually when trying to drag events on the master branch I get the following errors.

```
TypeError: Cannot read property 'def' of undefined
    at constructor.getEventDef (http://localhost:8100/build/vendor.js:162655:58)
    at constructor.Calendar.isEventInstanceGroupAllowed (http://localhost:8100/build/vendor.js:160163:36)
    at members.constructor.isEventInstanceGroupAllowed (http://localhost:8100/build/vendor.js:156649:24)
    at constructor.hitOver (http://localhost:8100/build/vendor.js:154024:29)
    at constructor.trigger (http://localhost:8100/build/vendor.js:152567:23)
    at constructor.handleHitOver (http://localhost:8100/build/vendor.js:152887:8)
    at constructor.handleDrag (http://localhost:8100/build/vendor.js:152868:10)
    at constructor.handleMove (http://localhost:8100/build/vendor.js:152461:9)
    at constructor.handleMouseMove (http://localhost:8100/build/vendor.js:152542:8)
    at members.constructor.proxy (http://localhost:8100/build/vendor.js:79922:14)

core.es5.js:1084 ERROR TypeError: Cannot read property 'start' of undefined
    at members.constructor.buildNewDateProfile (fullcalendar.js:13742)
    at members.constructor.mutateSingle (fullcalendar.js:13644)
    at fullcalendar.js:12343
    at Array.forEach (<anonymous>)
    at constructor.mutateEventsWithId (fullcalendar.js:12340)
    at constructor.reportEventDrop (fullcalendar.js:8529)
    at fullcalendar.js:4892
    at complete (fullcalendar.js:4086)
    at constructor.stop (fullcalendar.js:4106)
    at constructor.interactionEnd (fullcalendar.js:4884)
```

**Comment by arshaw** [comment]
Released in v3.5.0 ([blog post](https://fullcalendar.io/blog/2017/08/bootstrap-support-and-future-developments/#more-257))
